ocaml/xapi/extauth_plugin_ADwinbind.ml

```diff
@@ -554,7 +568,8 @@ let config_winbind_damon ~domain ~workgroup ~netbios_name =
       ; "winbind enum groups = no"
       ; "winbind enum users = no"
       ; Printf.sprintf "winbind cache time = %d" !Xapi_globs.winbind_cache_time
-      ; Printf.sprintf "machine password timeout = %d" !Xapi_globs.winbind_machine_pwd_timeout
+      ; Printf.sprintf "machine password timeout = %d"
```

Comment:
Stange, `make format` gives me different format, 
is there any conguration change recently about the format?

Reply:
Maybe the version of ocamlformat has changed, check with `ocamlformat -v` that 0.18.0 is being used